Desktop/html/js/quill.js

```diff
@@ -11005,12 +11005,10 @@ var FormulaBlot = function (_Embed) {
     value: function create(value) {
       var node = _get(FormulaBlot.__proto__ || Object.getPrototypeOf(FormulaBlot), 'create', this).call(this, value);
       if (typeof value === 'string') {
-        // Hack to replaces Katex with MathJax
-        node = MathJax.tex2svg(value, {display: false})
-        // window.katex.render(value, node, {
-        //   throwOnError: false,
-        //   errorColor: '#f00'
-        // });
+        window.katex.render(value, node, {
```

Comment:
Didnt we replace katex with mathjax for a reason?

Reply:
Well, I tried copy pasting to google docs and libre office and formula look fine so I guess it works with katex now.

Reply:
haha, no it's still work with mathjax,Because I made a simple modification in quill library (it is better not to modify the library directly), I restored the modification and made it again in an expanded format in jaspNote.js Yes, as I said, this PR will not add anything new or destroy the original feature.